# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sun-jdk-160/mime-type-remove.diff
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

we don't want java touching /etc/mailcap
Alejandro

we dont want java touching /usr/share/ registry files
mipe

we can't add new lines to .bin file because it does tail to checksum and to
generate tarball
Alejandro

Just a remark without modification: safe_mimetypeadd can not be used to add
mimetypes safely. This is done in postsysfiles.in. Create both mimetype and
mailcap entries.
Minto

--- jdk-6-beta-linux-i586.bin.orig	2006-03-12 09:41:15.000000000 +0100
+++ jdk-6-beta-linux-i586.bin	2006-03-12 09:53:57.000000000 +0100
@@ -147,7 +147,7 @@
     if [ -w `dirname ${MAILCAP_FILE}` ]; then
         MC_TEXT="${MC_TEXT}\n${MC_COMMENT}"
         MC_TEXT="${MC_TEXT}\n${MIME_TYPE}; $LATEST_JAVAWS_PATH %s"
-        echo ${echo_args}"${MC_TEXT}" > ${MAILCAP_FILE}
+        #echo ${echo_args}"${MC_TEXT}" > ${MAILCAP_FILE}
     else
         echo "WARNING - cannot write to file:"
         echo "       ${MAILCAP_FILE}"
@@ -171,9 +171,9 @@
     # Create the file if it does not exist
     if [ ! -w ${MIME_FILE} ]; then
         if [ -w `dirname ${MIME_FILE}` ]; then
-            echo "${NS_COMMENT1}"  > ${MIME_FILE}
-            echo "${NS_COMMENT2}" >> ${MIME_FILE}
-            echo "${NS_COMMENT3}" >> ${MIME_FILE}
+            #echo "${NS_COMMENT1}"  > ${MIME_FILE}
+            #echo "${NS_COMMENT2}" >> ${MIME_FILE}
+            #echo "${NS_COMMENT3}" >> ${MIME_FILE}
         else
             echo "WARNING - cannot write to file:"
             echo "       ${MIME_FILE}"
@@ -182,9 +182,9 @@
         fi
     fi
     # Add the jnlp entry if it does not already exist.
-    if [ -z "`grep ${MIME_TYPE} ${MIME_FILE}`" ]; then
-        echo ${JNLP_ENTRY} >> ${MIME_FILE}
-    fi
+    #if [ -z "`grep ${MIME_TYPE} ${MIME_FILE}`" ]; then
+    #    echo ${JNLP_ENTRY} >> ${MIME_FILE}
+    #fi
 }
 # End of UpdateMimeTypes
 
@@ -430,21 +430,21 @@
         fi
 
         # Setup the GNOME associations
-        AddGNOME application/java-archive \
-                 jar \
-                 java-archive \
-                 "${LATEST_JRE_PATH}/bin/java -jar" \
-                 sun-java \
-                 png \
-                 "Java Archive"
-
-        AddGNOME ${MIME_TYPE} \
-                 jnlp \
-                 java-web-start \
-                 "${LATEST_JAVAWS_PATH}" \
-                 sun-java \
-                 png \
-                 "Java Web Start Application"
+        #AddGNOME application/java-archive \
+        #         jar \
+        #         java-archive \
+        #         "${LATEST_JRE_PATH}/bin/java -jar" \
+        #         sun-java \
+        #         png \
+        #         "Java Archive"
+
+        #AddGNOME ${MIME_TYPE} \
+        #         jnlp \
+        #         java-web-start \
+        #         "${LATEST_JAVAWS_PATH}" \
+        #         sun-java \
+        #         png \
+        #         "Java Web Start Application"
     fi
 fi
 
